credentials/gobefore10.go

```diff
@@ -0,0 +1,30 @@
+// +build !go1.10
+
+/*
+ *
+ * Copyright 2020 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package credentials
+
+import (
+	"google.golang.org/grpc/grpclog"
+)
+
+func (t *TLSInfo) ParseSpiffeID() error {
+	grpclog.Info("go version prior to 1.10 doesn't support parsing URIs in certificates. Please consider a newer version")
```

Comment:
You can delete this; users don't need a log line every time they make a connection, and Go 1.9 isn't supported anyway.

Reply:
SG. I added a TODO above this function to remind me for the future deletion of this file. Please let me know if this works for you. Thanks!